packages/cozy-harvest-lib/src/components/TriggerManager.jsx

```diff
@@ -319,10 +252,12 @@ export class DumbTriggerManager extends Component {
    * @return {Object}          io.cozy.jobs document, runned with account data
    */
   async handleNewAccount(account) {
-    const trigger = await this.ensureTrigger(account)
+    const { client, trigger, konnector } = this.props
+    const trigger = await ensureTrigger(client, { trigger, account, konnector })
```

Comment:
I don't see `ensureTrigger` import. Is it missing?
(I bet it's fixed in a later commit, I'm currently reviewing https://github.com/cozy/cozy-libs/pull/956/commits/7cfcfa5e969172cceb2a17ac77458e7119cd380d)